web-common/src/features/feature-flags.ts

```diff
@@ -32,6 +32,14 @@ class FeatureFlags {
   readOnly = new FeatureFlag("rill", false);
 
   ai = new FeatureFlag("user", !import.meta.env.VITE_PLAYWRIGHT_TEST);
+  /**
+   * We make some calls to cloud for fetching user related stats. But this is not needed in playwring tests.
+   * While there wont be token in CI there could be one when running on a dev's laptop.
+   */
+  disableCloud = new FeatureFlag(
+    "user",
```

Comment:
This should be moved up to a "rill" feature flag because we don't anticipate users toggling this on themselves.